brigade.js

```diff
@@ -4,6 +4,8 @@ const projectOrg = "deislabs";
 const projectName = "cnab-go";
 
 const goImg = "golang:1.11";
+const goDind = "radumatei/golang-dind:1.11-dev"
```

Comment:
We should also track the Dockerfile for this image somewhere - it starts from DinD and adds Go 1.11.12.